client/components/Account/ChangeAccountNameForm.vue

```diff
@@ -0,0 +1,28 @@
+<!-- Form for changing account name (block style) -->
+
+<script lang="ts">
+import BlockForm from '@/components/common/BlockForm.vue';
+
+export default {
+  name: 'ChangeAccountNameForm',
+  mixins: [BlockForm],
+  data() {
+    return {
+      url: '/api/accounts',
+      method: 'PATCH',
+      hasBody: true,
+      setAccount: true,
+      fields: [
+        {id: 'name', label: 'Name', value: ''}
+      ],
+      title: 'Change account name',
```

Comment:
I was a bit confused here as well – I would make it explicit to mention to 'Change shared account name' (I thought I was, as a user, changing my own name)